TSRM/TSRM.c

```diff
@@ -16,13 +16,7 @@
 
 #include <stdio.h>
 #include <stdarg.h>
-
-#ifdef ZEND_DEBUG
-# include <assert.h>
-# define TSRM_ASSERT assert
-#else
-# define TSRM_ASSERT
-#endif
+#include "Zend/zend_portability.h"
```

Comment:
This is a bit dirty, in that TSRM is not supposed to depend on Zend... Though that separation probably is not particularly important nowadays anymore.

Reply:
We are already depending on ``ZEND_DEBUG``, or should that be an ``#ifdef TSRM_DEBUG``?

If we don't drop it, its definition should probably be changed to ``TSRM_ASSERT(c) assert(c)`` as otherwise we get the following compiler warning on release builds as can be seen from #5526 :
```
/home/vsts/work/1/s/TSRM/TSRM.c: In function ‘ts_free_thread’:
/home/vsts/work/1/s/TSRM/TSRM.c:482:14: warning: statement with no effect [-Wunused-value]
  TSRM_ASSERT(!in_main_thread);
             ~^~~~~~~~~~~~~~~~
```